### PR TITLE
Level feature & custom level feature added 

### PR DESCRIPTION
**Pull request by shaileshdeuex**
Level feature and Custom level feature added to the game and UI updated accordingly.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -3,49 +3,74 @@
...
+  font-size: 18px;
 }
 
 .btn {
   display: inline-block;
   padding: 0.50em 1.5em;
   border: 0.1em solid #FFFFFF;
   margin: auto 1px;
-  border-radius: 0.12em;
+  border-radius: 0.5em;
```
Comment (chirag-madlani) [inline src/App.css:83]: Any reason to use `em`

File: src/component/Board.jsx [hunk 2]
```diff
@@ -125,12 +147,18 @@ function Board() {
               src="https://giphy.com/embed/l0HlSDiA6WUytl9oA"
               className="winnerGif"
             ></iframe>
-            <button className="btn" onClick={restartGame}>
-              Play Again
-            </button>
+            <Button
+              className="btn"
```
Comment (chirag-madlani) [inline src/component/Board.jsx]: Component `Button` is not using this prop

File: src/component/Levels.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import Button from "./Button";
+
+function Levels({ levelSetUp, easy, medium, hard }) {
+  const [customeLevel, setcustomeLevel] = useState(easy);
```
Comment (chirag-madlani) [inline src/component/Levels.jsx]: What's need of level here?

We can pass it from parent since we are storing it there & also updating it

File: src/component/Board.jsx [hunk 4]
```diff
@@ -15,13 +17,18 @@ function isObjectEmpty(value) {
 }
 
 function Board() {
-  const cardLevel = 16;
-  const card = [];
+  // const cardLevel = 16;
+  let card = [];
```
Comment (chirag-madlani) [inline src/component/Board.jsx]: Can we avoid this ?

File: src/App.css (hunk 1 above)
Comment (shaileshdeuex) [reply src/App.css:83]: Not any specific need, just copied the style from the internet so I can give more focus on the Logic part.

File: src/component/Board.jsx (hunk 2 above)
Comment (shaileshdeuex) [reply src/component/Board.jsx]: Implemented.

File: src/component/Board.jsx (hunk 4 above)
Comment (shaileshdeuex) [reply src/component/Board.jsx]: Yes, I did const from let

File: src/component/Board.jsx [hunk 5]
```diff
@@ -62,22 +66,35 @@ function Board() {
...
   };
 
+  const scoreCalculation = () => {
+    baseScore = cardLevel / 2 - 1;
+    score = Math.round((baseScore / movesCounter) * 100);
+  };
+
   // function to check game is ended or not
   const gameEnd = () => {
+    scoreCalculation();
```
Comment (chirag-madlani) [inline src/component/Board.jsx]: Instead of storing it can we return from here & show it on UI?

File: src/component/Board.jsx [hunk 6]
```diff
@@ -62,22 +66,35 @@ function Board() {
...
+      }, 1500);
     }
   };
 
   // Starting game according to level or custome input
   const levelSetUp = (level) => {
     setCardLevel(level);
-    card = [];
+    // card = [];
+    card.length = 0;
```
Comment (chirag-madlani) [inline src/component/Board.jsx]: both are doing same thing, Why did you change `card = []` to ` card.length = 0`?

File: src/component/Board.jsx (hunk 5 above)
Comment (shaileshdeuex) [reply src/component/Board.jsx]: implemented

**Review by chirag-madlani — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]
